addons/mail/tests/common.py

```diff
@@ -349,19 +349,25 @@ def _find_mail_mail_wrecord(self, record, status=None, mail_message=None, author
             )
         return mail
 
-    def _find_sent_email(self, email_from, emails_to, subject=None):
-        """ Find an outgoing email based on from / to and optional subject when
-        having a conflict.
+    def _find_sent_email(self, email_from, emails_to, subject=None, body=None, attachment_names=None):
+        """ Find an outgoing email based on from / to and optional subject, body
+        and attachment names when having conflicts.
 
         :return sent_email: an outgoing email generated during the mock;
         """
         sent_emails = [
             mail for mail in self._mails
-            if set(mail['email_to']) == set(emails_to) and mail['email_from'] == email_from
+            if (set(email_normalize(email) for email in mail['email_to']) == set(email_normalize(email) for email in emails_to)
```

Comment:
Actually it is wanted to not do anything here, as we (or I) want to know what is stored at various steps. It allows to highlight that sometimes we store a normalized email, sometimes not, ... Can't you revert that change ? I would like to keep the asserts as dumb as possible (already complicated enough :D ) and let people call it with the right value.